bundlewrap/cmdline/parser.py

```diff
@@ -977,15 +964,12 @@ def build_parser_bw():
         action='store_true',
         dest='stdout_table',
         help=_("include command stdout in stats table"),
-    )
-    bw_run_p_default = int(environ.get("BW_NODE_WORKERS", "1"))
```

Comment:
To be discussed: Default was 1 here, but 4 everywhere else.

Reply:
I guess the reason for setting this to 1 was that you’d actually watch the output of `bw run`, like, a human reads it. It’ll be confusing if lots of different outputs are intermingled.

Personally, I never used `bw run` like this and I’m fine with settings this to `4` consistently.

@trehn Opinions on this?

Reply:
I assume most of the people using bw professionally will have set `BW_NODE_WORKERS` manually anyway, so this default won't apply. Personally i've always set `-p 1` manually if i need to see the output of running commands in order.